etc/cvmfs/config.d/alice.cern.ch.conf

```diff
@@ -0,0 +1,9 @@
+# The ALICE software framework uses many processes (not only multi-threading).
+# Each processes mapped libraries trigger open files to the cvmfs cache.
+# On many-core nodes, this can overflow the 128k default limit for cvmfs
+# processes.
+# See https://github.com/cvmfs/cvmfs/issues/3067 for details.
+CVMFS_NFILES=524288
```

Comment:
I suggest checking to make sure you're not decreasing the amount, only increasing.

Reply:
Good point, fixed.